src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,50 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "cinema_halls")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private int capacity;
```

Comment:
if we are using primitive fields for entity - than we should mark them with `@NotNull` constraint, or use wrappers as their types to avoid the null confusion